### DISCUSSION TITLE
Theme settings not saving

### DISCUSSION THREAD

**Discussion by Otts86** [open-webui/open-webui#5080] [General]
Hi all,

All of my settings seems to save except for my theme settings - it randomly reverts to the system theme.  I only have one user, me, but I access OWUI from multiple machines.  I've been meaning to see if others have the same issue.  Been using OWUI for 4+ months and this isn't a new behavior.

Thanks!

**Comment by justinh-rahb** [comment]
I believe this is one of the few settings that remains stored in LocalStorage, so if you've somehow got that blocked or automatically cleared between sessions, the setting will revert. It also doesn't transfer to other devices or browsers. I believe this is intentional so that you can set the theme according to your devices, as they may not all want the same. Myself I prefer the OLED Dark theme on mobile, but regular Dark otherwise for example.
